### DISCUSSION TITLE
Github layout seems to have gone sqwonk on Safari

### DISCUSSION THREAD

**Discussion by geoidesic** [community/community#31242] [Other Feature Feedback, Questions, & Ideas]
<img width="1624" alt="Screenshot 2022-08-29 at 17 13 20" src="https://user-images.githubusercontent.com/4637995/187246693-14825ba0-ae54-45b0-80db-079755bac0f7.png">

**Comment by airtower-luna** [comment]
Doesn't seem to be limited to Safari, I'm getting the same thing looking at your profile on Firefox. :fox_face: Interestingly not on my own profile, not sure what's the difference. Also, thank you for providing a giggle with the term "squonk". :smile_cat:

**Comment by cmilr** [comment]
I am having the same problem, but in Chrome--both Mac and Windows. All page content seems to have been shoved into the left side column.

<img width="605" alt="Screen Shot 2022-08-29 at 9 21 48 AM" src="https://user-images.githubusercontent.com/2286876/187248165-d1b6e890-009e-4749-9d4a-e4e854046b4a.png">

**Reply by geoidesic** [reply]
The joys of CI

**Comment by geoidesic** [comment]
Seems to have resolved itself?

**Reply by Chew** [reply]
I still experience this even with cache clear, unfortunately.

**Reply by geoidesic** [reply]
Oh yeah, me too. Seems to affect only some parts of the site. How do we report this or do the gods of github monitor these humble channels?

**Comment by geoidesic** [comment]
I reported the issue here: https://support.github.com/ticket/personal/0/1766774
